help/typing-notifications.md

```diff
@@ -2,12 +2,15 @@
 
 Zulip displays typing notifications when viewing a direct message or
 group direct message conversation to which one of the other
-participants is currently composing a message.
-
-Typing notifications are only sent while one is actively editing text
-in the compose box, and they disappear if typing is paused for about
-15 seconds.  Just having the compose box open will not send a typing
-notification.
+participants is currently composing a message. In the web and desktop apps,
+typing notifications are also displayed in the
+[all direct messages view](/help/direct-messages#view-all-direct-messages).
+
+Typing notifications are only sent while one is actively editing text in
+the compose box. They disappear if typing is paused for at least 5
+seconds, if all the content of the message is erased, or if the message is
```

Comment:
It might be better to just say "several" -- I am not sure this correctly summarizes the algorithm, and in any case it's probably more resolution than should be relevant to users.

Reply:
Yeah, that makes sense. Done!